gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/V2CanaryConfigController.groovy

```diff
@@ -50,13 +50,13 @@ class V2CanaryConfigController {
   @ApiOperation(value = "Create a canary configuration")
   @RequestMapping(method = RequestMethod.POST)
   Map createCanaryConfig(@RequestBody Map config) {
-    [id: canaryConfigService.createCanaryConfig(config)]
+    canaryConfigService.createCanaryConfig(config)
```

Comment:
Will need to make corresponding change deck for this one and the one just below.